docs/editors/new-editor.md

```diff
@@ -134,6 +134,14 @@ Possible values:
   Metals tries to fallback to `window/showMessageRequest` when possible.
 - `on`: the `metals/inputBox` request is fully supported.
 
+### `-Dmetals.pick-input`
```

Comment:
Related to some of the work I started in here: https://github.com/scalameta/metals/pull/1414, is it possible to not introduce this as a property, but rather as a `clientExperimentalCapabilities`? I feel like this would fit well there.

Reply:
Agreed, as I said in my previous comment, I would expect it to be a client config, rather then a server one. I've just repeated after `input-box`. If it's to be changed -- then this is to be changed as well :)

Reply:
Let's use the recently merged way defining capabilities.